docs/faq.md

```diff
@@ -18,7 +18,11 @@ The list of URLs is fed to our private instance of [WebPageTest](https://webpage
 
 As of March 1 2016, the tests are performed on Chrome for desktop and emulated Android (on Chrome) for mobile.
 
-The test agents are located in the [Internet Systems Consortium](https://www.isc.org/) data center in Redwood City, CA. Each URL is loaded 3 times with an empty cache ("first view"). The data from the median run (based on load time) is collected via a [HAR file](https://en.wikipedia.org/wiki/.har). The HTTP Archive collects these HAR files, parses them, and populates our database with the relevant information. The data is also available in [BigQuery](https://bigquery.cloud.google.com/dataset/httparchive:pages).
+The test agents are run from Google Cloud regions across the US. Each URL is loaded once with an empty cache ("first view") for normal metrics collection and again, in a clean browser profile, using [Lighthouse](https://developers.google.com/web/tools/lighthouse). The data is collected via a [HAR file](https://en.wikipedia.org/wiki/.har). The HTTP Archive collects these HAR files, parses them, and populates various tables in [BigQuery](https://bigquery.cloud.google.com/dataset/httparchive).
+
+The test infrastructure uses fixed, reserved, IP addresses and crawl traffic will come from one of these (and they will only be used for HTTP Archive crawl traffic):
+
+34.70.135.249, 34.122.58.12, 35.222.15.125, 34.148.244.203, 34.148.198.182, 35.194.67.240, 34.145.183.240, 34.105.27.245, 34.82.69.144, 35.236.8.58, 34.102.25.232, 34.106.20.144, 34.106.57.32, 34.125.80.148, 34.125.215.28
```

Comment:
Better to set as list?

```suggestion
- 34.70.135.249
- 34.122.58.12
- 35.222.15.125
- 34.148.244.203
- 34.148.198.182
- 35.194.67.240
- 34.145.183.240
- 34.105.27.245
- 34.82.69.144
- 35.236.8.58
- 34.102.25.232
- 34.106.20.144
- 34.106.57.32
- 34.125.80.148
- 34.125.215.28
```

Reply:
Was split on how easy it would be to copy/paste into ACL rules or if it would stretch out the FAQ page too much but I do prefer it as a list.